README.md

```diff
@@ -0,0 +1,16 @@
+# Tech Radar
+
+Bij Logius beheren wij een Tech Radar om onze API Standaarden team te helpen bijeenkomen op technologische keuzes. Het is grotendeels gebaseerd op [pioneering work by ThoughtWorks](https://www.thoughtworks.com/radar).
```

Comment:
```suggestion
Bij Logius beheren wij een Tech Radar om onze API Standaarden team te helpen bijeenkomen op technologische keuzes.
```
Wij zijn nog een stap verder verwijderd van Thoughtworks dan Zalando. Wellicht kan deze zin beter achterwege gelaten kunnen worden of Zalando vermelden; Zalando krijgt daarentegen al credit door middel van de aanwezigheid van de [licentietekst](https://github.com/Logius-standaarden/tech-radar/blob/main/LICENSE.txt).